### PR TITLE
Add link to new platform Github issue

### PR DESCRIPTION
**Pull request by janpio**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -92,7 +92,9 @@ If the above doesn't help, please [submit an issue](https://github.com/fastlane/
 
 ## System requirements
 
-Currently, _fastlane_ is officially supported to run on macOS. Support for other operating systems is limited and untested at this point in time. _fastlane_ uses system APIs that may not be implemented on other platforms, for example, we use the Ruby `fork` method for sub-process management, which isn't available on Windows.
+Currently, _fastlane_ is officially supported to run on 🍎 macOS.
```
Comment (KrauseFx) [inline docs/index.md]: Let's remove the apple here

File: docs/index.md [hunk 2]
```diff
@@ -92,7 +92,9 @@ If the above doesn't help, please [submit an issue](https://github.com/fastlane/
 
 ## System requirements
 
-Currently, _fastlane_ is officially supported to run on macOS. Support for other operating systems is limited and untested at this point in time. _fastlane_ uses system APIs that may not be implemented on other platforms, for example, we use the Ruby `fork` method for sub-process management, which isn't available on Windows.
+Currently, _fastlane_ is officially supported to run on 🍎 macOS. 
+
+But we are actively working on 🐧 Linux and 🖥️ Windows support for parts of _fastlane_. Some underlying software like Xcode or iTunes Transporter is only available for macOS, but many other tools and actions can theoretically also work on other platforms. Please see [this Github issue for further information](https://github.com/fastlane/fastlane/issues/11687).
```
Comment (KrauseFx) [inline docs/index.md]: Maybe we should not mention `actively`, as it might sound like that's our #1 priority

Comment (janpio) [reply docs/index.md]: Good point, I tried to avoid to create any expectations but missed this term. Removed it.

File: docs/index.md [hunk 3]
```diff
@@ -92,9 +92,9 @@ If the above doesn't help, please [submit an issue](https://github.com/fastlane/
 
 ## System requirements
 
-Currently, _fastlane_ is officially supported to run on 🍎 macOS.
```
Comment (revolter) [inline docs/index.md]: Why remove this?

Comment (janpio) [reply docs/index.md]: This specific one had the Apple which @KrauseFx didn't like.
But in the complete PR the line is still there, unchanged from the original.

Comment (revolter) [reply docs/index.md]: Sorry, didn't see his comment.

**Pull request merged**